### PR TITLE
 Configure and Restrict Sample Types and Pathogen Test Types for RSV …

### LINKED ISSUE #13158: Configure and Restrict Sample Types and Pathogen Test Types for RSV [0.5]
### Problem Description

For RSV cases, only specific Sample Types and Pathogen Test Types should be displayed and utilized based on the provided information. All other types that are not relevant for RSV should be hidden and unavailable for selection.

### Proposed Change

Adopt Relevant Sample and Pathogen Test Types:
- [ ] The relevant Sample Types and Pathogen Test Types for RSV should be integrated into the system, based on the information provided to us.
Restrict Unwanted Types:
- [ ] Sample Types and Pathogen Test Types that are not applicable to RSV should not be displayed in the system when managing RSV cases.

Pending Information:
- [ ] As we currently do not have the specific details regarding which types should be included or excluded for RSV, this issue is pending until that information is available.

### Added Value/Benefit

**Improved Data Accuracy:**
By only showing the relevant Sample Types and Pathogen Test Types for RSV, users can ensure that the correct information is entered. This reduces the risk of errors and ensures the data collected is specific and applicable to RSV case management.

### Acceptance Criteria

- [ ] Only the specified Sample Types and Pathogen Test Types relevant to RSV should be visible in the system for RSV cases.
- [ ] Any Sample Types and Pathogen Test Types not required for RSV should be hidden and unavailable for selection.
- [ ] Once the necessary information is received, the system should be updated accordingly.

### Implementation Details

_No response_

### Mockups

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by cazacmarin**
…#13158

Fixes #13158

### CODE REVIEW

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13175

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13175

File: sormas-api/src/main/java/de/symeda/sormas/api/sample/PathogenTestType.java [hunk 1]
```diff
@@ -24,38 +24,62 @@
 
 public enum PathogenTestType {
 
+	@Diseases(value = {
+			Disease.RESPIRATORY_SYNCYTIAL_VIRUS }, hide = true)
 	ANTIBODY_DETECTION,
 	ANTIGEN_DETECTION,
 	RAPID_TEST,
```
Comment (sergiupacurariu) [inline sormas-api/src/main/java/de/symeda/sormas/api/sample/PathogenTestType.java:31]: the changes for Pathogen Test Types cannot be seen on Pathogen test form

**Review by sergiupacurariu — APPROVED** [review]

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13175

**Pull request merged**